spec/client_spec.rb

```diff
@@ -693,7 +693,7 @@ def wait_global_key_missing(key)
 
   it 'Check attributes get_task_status' do
     task = @index.add_object!({ :name => "John Doe", :email => "john@doe.org" }, "1")
-    status = @index.get_task_status(task["taskID"])
+    status = @index.get_task_status(task["objectID"])
```

Comment:
This was actually correct, we what to get the status of the `taskID` not the `objectID`.